dev/docs/source/working_with_macros.rst

```diff
@@ -137,6 +140,20 @@ to format the XML for clarity::
    versions of Excel.
 
 
+Adding a VBA macro signature file to a libxlsxwriter file
```

Comment:
There is a typo here and below. It should say XlsxWriter.

Reply:
fixed in new commit